modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/user/custom_fields.jsp

```diff
@@ -18,8 +18,31 @@
 
 <%
 User selUser = (User)request.getAttribute(UsersAdminWebKeys.SELECTED_USER);
+
+PortletURL portletURL = PortletProviderUtil.getPortletURL(request, ExpandoColumn.class.getName(), PortletProvider.Action.MANAGE);
+
+portletURL.setParameter("modelResource", User.class.getName());
+portletURL.setParameter("redirect", themeDisplay.getURLCurrent());
```

Comment:
There're pre-defined objects available for use in JSP provided by the `defineObjects` tag, and current url is one of them. You can use the `currentURL` object instead of getting it from the themeDisplay.

To see a list of available objects provided by each of the `defineObjects` tag, check out the `DefineObjectsTag` classes. There're quite a few. Most frequently used ones are the following:

<liferay-frontend:defineObjects />
<liferay-theme:defineObjects />
<portlet:defineObjects />